classes/semver.js

```diff
@@ -16,7 +16,7 @@ class SemVer {
         version = version.version
       }
     } else if (typeof version !== 'string') {
-      throw new TypeError(`Invalid Version: ${require('util').inspect(version)}`)
+      throw new TypeError(`Invalid Version: ${version}`)
```

Comment:
```
> x = []
[]
> throw new Error(`invalid version ${x}`)
Uncaught Error: invalid version 
> 
```

This tells the end user nothing valuable if the version is not a string.

Reply:
How about `'Invalid Version. Must be a string. Got type "${typeof version}"'?

Reply:
Pushed something similar to this